pages/features.html

```diff
@@ -6,104 +6,134 @@
+  
+    <h1>{% trans "Customizable DJ Hardware Support" %}</h1>
+    {% trans "Free as in freedom as well as price, Mixxx plays nice with a variety of hardware without artificial restrictions or vendor lock-in mechanisms. The free version is the full version. Pick and choose gear from any source to build your unique setup." %}<!--TODO: Alignment somehow is messed up here.-->
+    
+    <div class="halfbox_left">
+      <div style="height: 200px;">
+        <center><img width="340px" src="{% static '/static/images/feature_midi2.png' %}" style="padding-top: 20px;"></center>
+      </div>
+
+      <h2>{% trans "MIDI and HID Controller Support" %}</h2>
+      <p>
+        {% blocktrans with hardware_compatibility_wiki_link="/wiki/doku.php/hardware_compatibility" %}<b>Included Presets</b><br> for controllers such as the Traktor Kontrol S4 MK2, Pioneer DDJ-SB &amp; DDJ-SB2, Allen &amp; Heath Xone K2, Hercules DJ Console Series, and many more. <a href="{{ hardware_compatibility_wiki_link }}">Full list of supported devices</a>{% endblocktrans %}
+      </p>
+
+      <p><b>{% trans "Programmable Mapping Engine" %}</b><br>
+        {% blocktrans with midi_scripting_wiki_link="/wiki/doku.php/midi_scripting" %}Cutting-edge JavaScript-based scripting engine provides maximum flexibility. MIDI or HID messages can be mapped onto custom functions which execute complex behaviour. <a href="{{ midi_scripting_wiki_link }}">Read more</a>{% endblocktrans %}
+      </p>
+    </div>
+    <div class="halfbox_right">
+      <div style="height: 200px;">
+        <center><img src="{% static '/static/images/feature_vinylcontrol.png' %}"></center>
+      </div>
 
-    <H1>{% trans "Powerful Library" %}</H1>
+      <h2>{% trans "Free Timecode Vinyl Control" %}</h2>
+      <p>
+        {% trans "Control your digital music files from turntables or CDJs and a mixer using timecoded vinyl or CDs. Mixxx is the only free timecode vinyl control software for Windows, Mac OS X, and Linux." %}
+      </p>
+
+      <p>
+        {% blocktrans with vinyl_control_manual_link="/manual/latest/chapters/vinyl_control.html" %}Serato CV 2, Serato CV 2.5, Traktor Scratch Mk1, and Mixvibes DVS V2 timecode are supported with absolute positioning, needle cueing, and end-of-record detection. Use any sound card(s) your OS supports to read timecode signals and play your music in high quality. <a href="{{ vinyl_control_manual_link }}">Read more</a>{% endblocktrans %}
+      </p>
+    </div>
+
+  <div style="clear: both;"></div>
+
+    <h1>{% trans "Sophisticated Music Library" %}</h1>
     <p>
-      {% trans "The Mixxx music library was designed from the ground up for DJs. Powered by a high-performance database, accessing and organizing your music is easy and intuitive." %}
+      {% trans "Stay organized and find the music you're looking for the moment you need it. Powered by a high-performance database, navigation and searching your collection is intuitive and fast." %}
```

Comment:
I'll try figuring out another way to include that sentiment with different wording.